nixos/modules/services/security/hockeypuck.nix

```diff
@@ -0,0 +1,73 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.services.hockeypuck;
+
+  configFile = pkgs.writeText "hockeypuck.config" ''
```

Comment:
Wow, that's super nice! Had no idea, I've implemented it now :+1:

Reply:
You should either set these values under `config.services.hockeypuck.settings` or use some recursive update function or else if anyone sets any value under `settings` it will erase your defaults.